### ISSUE TITLE
Please add AMP support

### ISSUE THREAD

**Issue by ram-you** [ekalinin/sitemap.js#89]
As indicated in this link: https://developers.google.com/search/docs/guides/create-URLs
We have to add entries for both canonical and **AMP URLs**.
It's a little similar to  **androidLink** support.

Just make those changes to sitemap.js:

``` javascript
...
+ this.ampLink = conf['ampLink'] || null;
...
-  var xml = '<url> {loc} {img} {lastmod} {changefreq} {priority} {links} {androidLink} {mobile} {news}</url>'
-    , props = ['loc', 'img', 'lastmod', 'changefreq', 'priority', 'links', 'androidLink', 'mobile', 'news']
+  var xml = '<url> {loc} {img} {lastmod} {changefreq} {priority} {links} {androidLink} {ampLink}  {mobile} {news}</url>'
+    , props = ['loc', 'img', 'lastmod', 'changefreq', 'priority', 'links', 'androidLink', 'ampLink', 'mobile', 'news']
...
+    } else if (this[p] && p == 'ampLink') {
+      xml = xml.replace('{' + p + '}', '<xhtml:link rel="amphtml" href="' + this[p] + '" />');
...

```

Thank you.

**Comment by nuclearspike** [comment]
+1

**Comment by nuclearspike** [comment]
Sweet! Thanks! Is there a schedule for when this will be in the NPM package or should I just ref the git repo in my package.json?

**Comment by ekalinin** [comment]
> Is there a schedule for when this will be in the NPM package

Just published it — [1.9.0](https://github.com/ekalinin/sitemap.js/releases/tag/1.9.0)

**Comment by nuclearspike** [comment]
Sweet! Thanks for publishing!
